sidekiq_publisher.gemspec

```diff
@@ -47,9 +47,9 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "appraisal"
   spec.add_development_dependency "bundler"
   spec.add_development_dependency "database_cleaner"
-  spec.add_development_dependency "ddtrace", ">= 0.39.0"
+  spec.add_development_dependency "ddtrace"
```

Comment:
Should we add an updated version requirement here?

```suggestion
  spec.add_development_dependency "ddtrace", "> 1.8.0"
```

Reply:
yes! good catch